bootstrap.php

```diff
@@ -1,37 +1,37 @@
 <?php
-use Doctrine\ORM\Tools\Setup,
-    Doctrine\ORM\EntityManager,
-    Doctrine\Common\EventManager as EventManager,
-    Doctrine\ORM\Events,
-    Doctrine\ORM\Configuration,
-    Doctrine\Common\Cache\ArrayCache as Cache,
-    Doctrine\Common\Annotations\AnnotationRegistry, 
-    Doctrine\Common\Annotations\AnnotationReader,
-    Doctrine\Common\ClassLoader;
 
 $loader = require __DIR__.'/vendor/autoload.php';
-$loader->add('Skel', __DIR__.'/src');
-// $loader->add('Coderockr', __DIR__.'/../coderockr/soa/src');
+\Doctrine\Common\Annotations\AnnotationRegistry::registerLoader(array($loader, 'loadClass'));
 
-//doctrine
-$config = new Configuration();
-//$cache = new Cache();
-$cache = new \Doctrine\Common\Cache\ApcCache();
-$config->setQueryCacheImpl($cache);
-$config->setProxyDir('/tmp');
-$config->setProxyNamespace('EntityProxy');
-$config->setAutoGenerateProxyClasses(true);
- 
-//mapping (example uses annotations, could be any of XML/YAML or plain PHP)
-AnnotationRegistry::registerFile(__DIR__. DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'doctrine' . DIRECTORY_SEPARATOR . 'orm' . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'Doctrine' . DIRECTORY_SEPARATOR . 'ORM' . DIRECTORY_SEPARATOR . 'Mapping' . DIRECTORY_SEPARATOR . 'Driver' . DIRECTORY_SEPARATOR . 'DoctrineAnnotations.php');
+use Silex\Application;
+use Silex\Provider\TwigServiceProvider;
+use Silex\Provider\MonologServiceProvider;
+use Silex\Provider\DoctrineServiceProvider;
+use DerAlex\Silex\YamlConfigServiceProvider;
+use MJanssen\Provider\RoutingServiceProvider;
+use Dflydev\Silex\Provider\DoctrineOrm\DoctrineOrmServiceProvider;
 
-\Doctrine\Common\Annotations\AnnotationRegistry::registerFile(
-    __DIR__ . '/vendor/jms/serializer/src/JMS/Serializer/Annotation/Type.php'
-);
+// Decline static file requests back to the PHP built-in webserver
+if (php_sapi_name() === 'cli-server') {
```

Comment:
Se for linha de comando vai ser sempre "development"? E se executar o comando no servidor de produção? Não é melhor respeitar a variável de ambiente existente, quando existe?